taskcluster/test-evaluate_tflite-linux-amd64-py36m-opt.yml

```diff
@@ -0,0 +1,13 @@
+build:
+  template_file: test-linux-opt-base.tyml
+  dependencies:
+    - "linux-amd64-cpu-opt"
+    - "test-training_16k-linux-amd64-py36m-opt"
```

Comment:
Looks like you are missing `test_model_task` field here :)

Reply:
Thanks :)